change/@ni-nimble-components-376b2ed8-09ef-409f-9acb-e7f04b283d39.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Any specs that need updating to capture this concept?

Reply:
Added some docs to the table data API spec.